ipaserver/install/server/upgrade.py

```diff
@@ -1692,11 +1695,26 @@ def upgrade_configuration():
             )
         upgrade_pki(ca, fstore)
 
+        if kra.is_configured():
+            logger.info('[Ensuring ephemeralRequest is enabled in KRA]')
+            cainstance.backup_config(path=paths.KRA_CS_CFG_PATH)
```

Comment:
I think it would be more straightforward to have `krainstance.backup_config()` instead of `cainstance.backup_config(new_arg_with_default_value)`.  The common bits can be extracted to a method in the shared `DogtagInstance` base class.

Reply:
I laid the groundwork for this in a separate patch. The changes are small but required significant reformatting to get past the pep8 checks.